src/app/payment/creditCard/configureCardValidator.ts

```diff
@@ -16,4 +16,117 @@ export default function configureCardValidator(): void {
             [810, 817],
         ],
     });
+
+    // Add support for Mada-BIN Cards.
+    creditCardType.addCard({
+        niceType: 'Mada',
+        type: 'mada',
+        patterns: [
```

Comment:
any way we can sort these numerically, and also put several on a line in a nice way, so it isn't so long?

Reply:
Let me change that

Reply:
@ejesinskyBC done

Reply:
You can go ahead and try that if the lint/etc let's it pass, though I was referring to like 4/5 numbers per line in order to retain a reasonable line length, for example,
`[
400861, 401757, 407197, 407395, 409201,
410685, 412565, 417633, 419593, 422817,
422818, 422819, 428331, 428671, 428672,
428673, 431361, 432328, 434107, 439954,
440533, 440647, 440795, 445564, 446393,
446404, 446672, 455036, 455708, 457865,
458456, 462220, 468540, 468541, 468542,
468543, 483010, 483011, 483012, 484783,
486094, 486095, 486096, 489317, 489318,
489319, 493428, 504300, 506968, 508160,
513213, 520058, 521076, 524130, 524514,
529415, 529741, 530060, 530906, 531095,
531196, 532013, 535825, 535989, 536023,
537767, 539931, 543085, 543357, 549760,
554180, 557606, 558848, 585265, 588845,
588846, 588847, 588848, 588849, 588850,
588851, 588982, 588983, 589005, 589206,
604906, 605141, 636120, 968201, 968202,
968203, 968204, 968205, 968206, 968207,
968208, 968209, 968210, 968211]
`

Reply:
@ejesinskyBC done